### PR TITLE
Voleyko/task 2

### PR DESCRIPTION
**Pull request by VoleykoN**
Implemented the task 2(var1)

### CODE REVIEW

File: Tasks/Voleyko/Task2.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+13:'тренадцать минут',14:'четырнадцать минут',15:'пятнадцать минут',16:'шестьнадцать минут',17:'семьнадцать минут',18:'восемьнадцать минут',19:'девятнадцать минут',20:'двадцать минут',21:'двадцать одна минута',22:'двадцать две минуты',23:'двадцать три минуты',24:'двадцать четыре минуты',
+25:'двадцать пять минут',26:'двадцать шесть минут',27:'двадцать семь минут',28:'двадцать восемь минут',29:'двадцать девять минут',30:'тридцать минут',31:'дридцать одна минута',32:'тридцать две минуты',33:'тридцать три минуты',34:'тридцать четыре минут',35:'тридцать пять минут',36:'тридцать шесть минут',37:'тридцать семь минут',38:'тридцать восемь минут',
+39:'тридцать девять минут',40:'сорок минут',41:'сорок одна минута',42:'сорок две минуты',43:'сорок три минуты',44:'сорок четыре минуты',45:'сорок пять минут',46:'сорок шесть минут',47:'сорок семь минут',48:'сорок восемь минут',49:'сорок девять минут',50:'пятьдесят минут',51:'петьдесят одна минута',52:'пятьдесят две минуты',53:'пятьдесят три минуты',54:'пятьдесят четыре минуты',55:'пятьдесят пять минут',
+56:'пятьдесят шесть минут', 57:'пятьдесят семь минут',58:'пятьдесят восемь минут',59:'пятьдесят девять минут'}
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Voleyko/Task2.py:12]: I've already seen such variable names somewhere =) 

Please make sure to use more meaningful names in the future, the data structure is good enough, but it's possible to make it better by eliminating the minutes tokens

File: Tasks/Voleyko/Task2.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+print('Приветствуем в нашей программе:\nНажмите 1-Чтобы самостоятельно ввести время,\nНажмите 2-Чтобы текущее время считалось автоматически')
+
+start = int(input())
+
+if start == 1:
+    user_h = input('Пожалуйста,введите час:')
+    user_m = input('Пожалуйста, ввевиде минуты:')
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Voleyko/Task2.py:20]: a user supposed to input some string in "HH:MM" format

**Review by MikitaTsiarentsyeu — APPROVED** [review]
Good enough to satisfy the tasks requirements, also the data structure could be simplified. Your score is 9.

**Comment by VoleykoN** [comment]
> Good enough to satisfy the tasks requirements, also the data structure could be simplified. Your score is 9.

Thank you! I'm very glad to see this score, because I thought I didn't make it 100%. I wanted to do another one version task_2 more simply and minimalistic.

**Pull request merged**